Examples/Tests/LoadExternalField/inputs_rz

```diff
@@ -55,14 +55,11 @@ proton.do_not_deposit = 1
 proton.mass = m_p
 proton.charge = q_e
 
-#electron.injection_style = "SingleParticle"
-#electron.single_particle_pos = 0.0  0.2  2.5
-#electron.single_particle_u = 0.0  0.0  0.0
-#electron.single_particle_weight = 1.0
-#electron.mass = 1.0
-#electron.charge = -q_e*1.0e-20
-
 # Diagnostics
-diagnostics.diags_names = diag1
```

Comment:
This test is now a restart test ; as a consequence, the checkpoints need to be saved.